### PR TITLE
[IMP] account_edi, l10n_in_edi: onboarding view improvements

### PR DESCRIPTION
**Pull request by suth-odoo**
- Renamed E-invoice(IN) into GST E-invoice (India)
- Moved the process now button to right side
- Added decoration colors to Electronic invoicing based on edi_blocking_level

task-3659614

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_in_edi/data/account_edi_data.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="edi_in_einvoice_json_1_03" model="account.edi.format">
-        <field name="name">E-Invoice (IN)</field>
+        <field name="name">GST E-Invoice (India)</field>
```
Comment (hamo-odoo) [inline addons/l10n_in_edi/data/account_edi_data.xml:4]: May be a different commit message for changes in `l10n_in_edi`? Doesn't related to `account_edi`

**Review by hamo-odoo — APPROVED** [review]
@kais-odoo @suth-odoo 
Can you please reword the commit title?
`account_edi` doesn't relate to GST E-Invoice
May be `[IMP] account_edi: on boarding view improvements` or something better?

@jco-odoo Changes LGTM

**Comment by jco-odoo** [comment]
@robodoo r+ rebase-ff
